src/utils/index.js

```diff
@@ -14,6 +14,14 @@ export const queryAll = (selector, context = document) => {
 	return Array.from(context.querySelectorAll(selector));
 };
 
+export const nextTick = (callback) => {
+	requestAnimationFrame(() => {
+		requestAnimationFrame(() => {
+			callback();
+		});
+	});
+};
```

Comment:
Love this! 

I was seeing it through the code and was preparing to comment `requestAnimationFrame` being a bit dangerous for such a case as things can manage to happen in one "tick" even if you don't want to, but I see you've got it covered with two. 😄

Reply:
Yeah, learned that the hard way 🤠